site/ocrd-events.json

```diff
@@ -1,36 +1,4 @@
 [
-  {
-    "title": "Kitodo-Anwendertreffen",
-	"color": "black",
-    "rrule": {
-      "dtstart": "2021-11-24T11:00:00",
-      "tzid": "Europe/Berlin"
-    }
-  },
-    {
-    "title": "Kitodo-Anwendertreffen",
-	"color": "black",
-    "rrule": {
-      "dtstart": "2021-11-25T09:00:00",
-      "tzid": "Europe/Berlin"
-    }
-  },
-     {
-    "title": "vBIB21",
-	"color": "black",
-    "rrule": {
-      "dtstart": "2021-12-01T10:00:00",
-      "tzid": "Europe/Berlin"
-    }
-  },
-       {
-    "title": "vBIB21",
-	"color": "black",
-    "rrule": {
-      "dtstart": "2021-12-02T10:00:00",
-      "tzid": "Europe/Berlin"
-    }
-  },
```

Comment:
These aren't actually deleted, just moved down, right?

Reply:
yes, I only deleted them from the left column. It's not bad to have the other events in the calendar, I guess